### PR TITLE
Update response type names

### PR DESCRIPTION
**Pull request by culli**
This updates type names to match recent changes in the underlying OpenAPI definitions.

### CODE REVIEW

**Comment by kvindascr** [comment]
@culli this PR is not compiling.
Is that caused because it depends on the other PR?

File: src/wrapper/AirtopSessions.ts [hunk 1]
```diff
@@ -31,14 +31,14 @@ export class AirtopSessions {
   list(
     request?: Airtop.SessionsListRequest,
     requestOptions?: SessionsNamespace.RequestOptions,
-  ): Promise<Airtop.SessionsWithPaginationEnvelopeDefaultMetaWrapper> {
+  ): Promise<Airtop.SessionsResponse> {
```
Comment (kvindascr) [inline src/wrapper/AirtopSessions.ts:34]: I was wondering if we should just remove the types and let typescript deduct them from whatever is returned from sessions.list in this example.

What do you think?

Comment (culli) [reply src/wrapper/AirtopSessions.ts:34]: That would be better for maintenance here, for sure.

**Comment by culli** [comment]
> @culli this PR is not compiling. Is that caused because it depends on the other PR?

That is true.  We'll have to improve this process.

**Pull request merged**